core/src/main/resources/oracle/weblogic/deploy/aliases/category_modules/Log.json

```diff
@@ -0,0 +1,51 @@
+        "RedirectStderrToServerLogEnabled":              [ {"version": "[10,)",     "wlst_mode": "both", "wlst_name": "RedirectStderrToServerLogEnabled",              "wlst_path": "WP001", "value": {"default": "false" }, "wlst_type": "boolean"} ],
+        "RedirectStdoutToServerLogEnabled":              [ {"version": "[10,)",     "wlst_mode": "both", "wlst_name": "RedirectStdoutToServerLogEnabled",              "wlst_path": "WP001", "value": {"default": "false" }, "wlst_type": "boolean"} ],
+        "RotateLogOnStartup":                            [ {"version": "[10,)",     "wlst_mode": "both", "wlst_name": "RotateLogOnStartup",                            "wlst_path": "WP001", "value": {"default": "true"  }, "wlst_type": "boolean"} ],
+        "RotationTime":                                  [ {"version": "[10,)",     "wlst_mode": "both", "wlst_name": "RotationTime",                                  "wlst_path": "WP001", "value": {"default": "00:00" }, "wlst_type": "string" } ],
+        "RotationType":                                  [ {"version": "[10,)",     "wlst_mode": "both", "wlst_name": "RotationType",                                  "wlst_path": "WP001", "value": {"default": "bySize"}, "wlst_type": "string" } ],
+        "ServerLoggingBridgeUseParentLoggersEnabled":    [ {"version": "[10,)",     "wlst_mode": "both", "wlst_name": "ServerLoggingBridgeUseParentLoggersEnabled",    "wlst_path": "WP001", "value": {"default": "false" }, "wlst_type": "boolean"} ],
+        "StacktraceDepth":                               [ {"version": "[10,)",     "wlst_mode": "both", "wlst_name": "StacktraceDepth",                               "wlst_path": "WP001", "value": {"default": 5       }, "wlst_type": "integer"} ],
+        "StdoutFilter":                                  [ {"version": "[10,)",     "wlst_mode": "both", "wlst_name": "StdoutFilter",                                  "wlst_path": "WP001", "value": {"default": "None"  }, "wlst_type": "string" } ],
+        "StdoutFormat":                                  [ {"version": "[10,)",     "wlst_mode": "both", "wlst_name": "StdoutFormat",                                  "wlst_path": "WP001", "value": {"default": "standard"}, "wlst_type": "string" } ],
+        "StdoutLogStack":                                [ {"version": "[10,)",     "wlst_mode": "both", "wlst_name": "StdoutLogStack",                                "wlst_path": "WP001", "value": {"default": "true"  }, "wlst_type": "boolean"} ],
+        "StdoutSeverity":                                [ {"version": "[10,)",     "wlst_mode": "both", "wlst_name": "StdoutSeverity",                                "wlst_path": "WP001", "value": {"default": "Notice"}, "wlst_type": "string" } ]
+    },
+    "wlst_attributes_path": "WP001",
+    "wlst_paths": {
+        "WP001": "/Log/%DOMAIN%"
```

Comment:
Shouldn't this be /Log${:s}/%DOMAIN%

Reply:
Nope. It (the /Log folder) is only visible in WLST in ONLINE mode, and then the folder name is Log...not Logs